merkle/src/hash.rs

```diff
@@ -116,7 +116,6 @@ where
     T: Clone + AsRef<[u8]>,
 {
     /// Returns the hash value for the data stream written so far.
-    #[inline]
```

Comment:
Why removing this?

Reply:
Because `inline` only has an effect on functions with a body, and clippy reminded me to fix it